lib/inc/facter/util/environment.hpp

```diff
@@ -1,70 +0,0 @@
-/**
```

Comment:
In the process of putting this back (along with its .cc files) since it was in inc/facter